client/src/api/v2/camera.ts

```diff
@@ -240,10 +240,10 @@ function createStatusPayloadHook<T, U>(
   };
 }
 
-const CameraRecordingStatusPayloads = Partial({
-  primary: CameraRecordingStatusPayload,
-  secondary: CameraRecordingStatusPayload,
-});
+// const CameraRecordingStatusPayloads = Partial({
+//   primary: CameraRecordingStatusPayload,
+//   secondary: CameraRecordingStatusPayload,
+// });
```

Comment:
If we no longer need this code we should remove it entirely :)

Reply:
I wasn't sure if the hooks were intended to return the status payloads for both devices or just for the specified device. The latter is consistent with how it was using in `CameraRecordingStatus` but I vaguely recall discussing the former.

Reply:
Do whatever you think is neatest :) I don't think I have a big investment in either (although I haven't had much of a dig through the code recently)

Reply:
done
c294012